app/src/main/java/mozilla/lockbox/presenter/ItemListPresenter.kt

```diff
@@ -70,4 +96,13 @@ class ItemListPresenter(
 
         dispatcher.dispatch(action)
     }
+}
+
+fun Observable<Pair<List<ItemViewModel>, ItemListSort>>.sort(): Observable<List<ItemViewModel>> {
+    return this.map { pair ->
+        when (pair.second) {
+            ItemListSort.ALPHABETICALLY -> { pair.first.sortedBy { it.title } }
+            ItemListSort.RECENTLY_USED -> { pair.first.sortedBy { it.timeLastUsed }}
+        }
+    }
```

Comment:
I miscommunicated: this shouldn't be an extension method on `Observable`, this should be inlined in `ItemListPresenter`.